include/xgboost/learner.h

```diff
@@ -274,9 +281,16 @@ class Learner : public Model, public Configurable, public dmlc::Serializable {
 };
 
 struct LearnerModelParamLegacy;
+/**
+ * \brief Strategy for building multi-target models.
+ */
+enum class Strategy : std::int32_t {
+  kComposite = 0,
+  kMono = 1,
+};
```

Comment:
```suggestion
enum class MultiTargetStrategy : std::int32_t {
  kComposite = 0,
  kMonolithic = 1,
};
```

Let's be more specific in the name